src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -323,7 +323,7 @@ public sealed override bool HasInitializer
             get { return _hasInitializer; }
         }
 
-        public VariableDeclaratorSyntax VariableDeclaratorNode
+        private VariableDeclaratorSyntax VariableDeclaratorNode
```

Comment:
A private member is an explicit contract with code within its containing type.

---
In reply to: [80850539](https://github.com/dotnet/roslyn/pull/13999#discussion_r80850539) [](ancestors = 80850539,80816186,80815616)